docs/configuration/configuration.md

```diff
@@ -2648,6 +2648,28 @@ relabel_configs:
   [ - <relabel_config> ... ]
 ```
 
+### `<tracing>`
+`tracing` configures exporting traces from Prometheus to a tracing backend via the OTLP protocol.
+
+Must be used together with `--enable-feature=tracing` flag.
+
+```yaml
+# Decides whether to use HTTP or gRPC client to export the traces.
+[ client_type: <string> | default = grpc ]
+
+# Service name to be used for tracing purposes.
+[ service_name: <string> | default = prometheus ]
+
+# Endpoint to send the traces to. By default, this is the Otel Collector's gRPC port 7314 on a localhost.
+[ endpoint: <string> | default = localhost:7314 ]
+
+# Sets the probability a given trace will be sampled. Must be a float between 0 and 1. The default is 0, meaning no traces are exported.
```

Comment:
The range is inclusive of 0 and 1 https://pkg.go.dev/go.opentelemetry.io/otel/sdk@v1.2.0/trace#TraceIDRatioBased.

```suggestion
# Sets the probability a given trace will be sampled. Must be a float from 0 through 1. The default is 0, meaning no traces are exported.
```

Reply:
As pointed out by @roidelapluie, having a sampling fraction of 0 means Prometheus won't create any new traces but still continue/export existing traces from other services.

```suggestion
# Sets the probability a given trace will be sampled. Must be a float between 0 and 1.
```

Reply:
Agreed, that's a more precise statement on behavior, I'll remove the second sentence.